src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -201,29 +256,37 @@ def output(self, index):
     def mark_node(self, node):
         return node
 
+    def try_decode_get_attr(self):
+        pt_value = get_value_from_getattr(self.graph_element, self.pt_module)
+        assert pt_value is not None, "Couldn't retrieve value from prim::GetAttr"
+        if not isinstance(pt_value, torch.jit.ScriptModule) or isinstance(pt_value, torch.jit.TracedModule):
+            return ivalue_to_constant(pt_value)
+        else:
+            return []
```

Comment:
It will always lead to assertion failure later in the C++ code. So why not just always treat `GetAttr` as a constant using `as_constant_tensor` and move all associated code to that method? I mean just technically, would it be just simpler to implement? In this case we avoid creation this `try_decode_get_attr` decoder API entry point.

Reply:
On the C++ side, in `op_table` we just add a new mapping from `GetAttr` to `translate_constant` if we follow this approach. Please let me know what I'm missing here.

Reply:
The problem here is the fact that GetAttr can return not only constant, but also a module. We can do with that case nothing, so we need a way to fail somehow, it can be exception, but I decided to return empty list

Reply:
The disadvantage is we lose control over how to treat unfrozen graphs -- they are always viewed as frozen.

Reply:
This function not only call `ivalue_to_constant` it also calls `get_value_from_getattr` it can't be treated the same as simple constant translation.

Reply:
The same logic can be implemented in `as_constant_tensor` -- just read op type and if it is `GetAttr` call this code.

Reply:
Let us leave it as-is because indeed benefits are not obvious and the final way how we are going to make it in the real product depends on our involving in training.